op-batcher/batcher/channel_builder_test.go

```diff
@@ -32,20 +33,30 @@ var defaultTestChannelConfig = ChannelConfig{
 	ApproxComprRatio:   0.4,
 }
 
+// TestNonZeroMaxFrameSize validates that the [ChannelConfig]
+// will not allow a zero max frame size when checked.
+func TestNonZeroMaxFrameSize(t *testing.T) {
```

Comment:
In general test names should mention the type (and method, if there's a specific one) that is being tested, e.g. here
`TestChannelConfig_Check` and then you can add individual sub-tests with `t.Run`.

Reply:
Also, bumping my previous comment, this is a good area for table tests. There is a specific input (channel config), the exact same function being called, & then a deterministic output.